### DISCUSSION TITLE
Can you higlight text in Markdown?

### DISCUSSION THREAD

**Discussion by akim67212** [community/community#143143] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I was wondering, is it possible to highlight syntax in a Markdown file? I want to highlight different syntaxes that I have used in SQL.

**Comment by SatyamPote** [comment]
==This text is highlighted==
<mark>This text is highlighted</mark>


if answer is write the mark as correct

**Reply by akim67212** [reply]
@SatyamPote Is it possible to highlight the text in a light grey color?

**Comment by akim67212** [comment]
Is it possible to highlight the text a light grey color?
